### ISSUE TITLE
how do i prevent auto jshint configuration everytime i save?

### ISSUE THREAD

**Issue by shaunstanislaus** [SublimeLinter/SublimeLinter-jscs#34]
how do i prevent auto jshint configuration everytime i save?

No configuration found. Add a .jscsrc file to your project root or use the -c option
<img width="1440" alt="screen shot 2015-07-15 at 18 26 21" src="https://cloud.githubusercontent.com/assets/1703530/8729952/b21a1aea-2c20-11e5-8666-e26ad1927b6e.png">

**Comment by shaunstanislaus** [comment]
can anyone reply me?

**Comment by robwise** [comment]
@shaunstanislaus JSCS needs to know what you want it to check for, so either create a `.jscsrc` file like it says ([options here](http://jscs.info/overview.html#options)) or you can just go into your SublimeLinter settings and have it pass a configuration preset via a command-line argument ([presets here](http://jscs.info/overview.html#presets)). So for example:

```json
// SublimeLinter.sublime-settings
"linters":
{
    "jscs": 
    {
        "@disable": false,
        "args": [ "--preset=airbnb", "--esnext"],
        "excludes": []
    }
}
```

**Comment by zxqfox** [comment]
Just fyi: airbnb preset has esnext flag itself ;-)

https://github.com/jscs-dev/node-jscs/blob/master/presets/airbnb.json#L2

**Comment by shaunstanislaus** [comment]
hey zxqfox,

what do you mean that airbnb preset has esnext flag itself?

do you mean that in the args, i should remove --esnext cause preset=airbnb already has the setting preset?

**Comment by zxqfox** [comment]
Yeah, I mean you don't need to manually set --esnext flag when using airbnb.

**Comment by shaunstanislaus** [comment]
oh okay thank you (:

could you help me out in this issue please? ):
```
https://github.com/CraigWilliams/BeautifyRuby/issues/97
```
